### PR TITLE
:spades: Extract environment stuff

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const dotenv = require('dotenv');
+const requireEnv = require('require-environment-variables');
+
+function validate() {
```
Comment (neilbmclaughlin) [inline config/environment.js]: Strictly speaking this is configuring (the 2nd part is configuring validation).

Bit pedantic but then so is lint :)

Also, giving the new GitHub review process a spin.

Comment (st3v3nhunt) [inline config/environment.js]: So it is. What change(s) do you suggest?

Comment (neilbmclaughlin) [inline config/environment.js]: Rename to configure()?

[Note: had to do a manual PR page refresh for your comment to appear. Was aware of your comment through Slack]

Comment (st3v3nhunt) [inline config/environment.js]: Done

**Pull request merged**